src/server.ts

```diff
@@ -4,31 +4,221 @@ import express from "express";
 import { Client } from "pg";
 import { getEnvVarOrFail } from "./support/envVarUtils";
 import { setupDBClientConfig } from "./support/setupDBClientConfig";
+import { EmbedBuilder, WebhookClient } from "discord.js";
 
-dotenv.config(); //Read .env file lines as though they were env vars.
+//-------------------------------------------------------------------Read .env file lines as though they were env vars.
+dotenv.config();
+
+const id = process.env.DISCORD_ID ?? "null";
+const token = process.env.DISCORD_TOKEN ?? "null";
+
+const webhookClient = new WebhookClient({
+  id: id,
+  token: token,
+});
+
+export const url = "https://new-resource.netlify.app";
+
+// export const url =
+//   process.env.NODE_ENV === "production"
+//     ? "https://new-resource.netlify.app/"
+//     : "http://localhost:4000";
 
 const dbClientConfig = setupDBClientConfig();
 const client = new Client(dbClientConfig);
 
-//Configure express routes
+//-------------------------------------------------------------------Configure express routes
 const app = express();
 
-app.use(express.json()); //add JSON body parser to each following route handler
-app.use(cors()); //add CORS support to each following route handler
+//-------------------------------------------------------------------add JSON body parser to each following route handler
+app.use(express.json());
+
+//-------------------------------------------------------------------add CORS support to each following route handler
+app.use(cors());
 
+//-------------------------------------------------------------------GET requests
+//-------------------------------------------------------------------Default response
 app.get("/", async (req, res) => {
   res.json({ msg: "Hello! There's nothing interesting for GET /" });
 });
 
-app.get("/health-check", async (req, res) => {
+//-------------------------------------------------------------------Get list of all resources
+app.get("/resources", async (req, res) => {
+  try {
+    const query = "SELECT * FROM resources ORDER BY resource_id DESC";
+    const response = await client.query(query);
+    res.status(200).send(response.rows);
+  } catch (err) {
+    console.error(err);
+  }
+});
+
+//-------------------------------------------------------------------Get all users
+app.get("/users", async (req, res) => {
+  try {
+    const query = "SELECT * FROM users";
+    const response = await client.query(query);
+    res.status(200).send(response.rows);
+  } catch (err) {
+    console.error(err);
+  }
+});
+
+//-------------------------------------------------------------------Get all likes for a specific post
+app.get("/resources/:resourceID/likes", async (req, res) => {
+  const resourceID = req.params.resourceID;
```

Comment:
This is a very small thing, but because everything else is so clean, we're being nit-picky ;) ---- this particular line is not quite consistent with your subsequent queries, as it uses req.params.resourceID, as opposed to { resourceId } = ...